climada/hazard/trop_cyclone.py

```diff
@@ -515,25 +542,14 @@ def _close_centroids(t_lat, t_lon, centroids, buffer=CENTR_NODE_MAX_DIST_DEG):
     eye_bounds = np.c_[t_lon, t_lat, t_lon, t_lat]
     eye_bounds[:,:2] -= buffer
     eye_bounds[:,2:] += buffer
-
-    # rectangle crosses antimeridian:
-    eye_bounds[eye_bounds[:,0] <= -180,0] += 360
-    eye_bounds[eye_bounds[:,2] > 180,2] -= 360
-
     centr_lat, centr_lon = centroids[:, 0], centroids[:, 1]
-    msk_lat = ((eye_bounds[:,None,1] < centr_lat[None])
-               & (centr_lat[None] < eye_bounds[:,None,3]))
-
-    # rectangle might cross antimeridian or not
-    msk_lon_crosses = (eye_bounds[:,2] < eye_bounds[:,0])
-    msk_lon_within = ~msk_lon_crosses
-    msk_lon = (eye_bounds[:,None,0] < centr_lon[None])
-    msk_lon_upper = (centr_lon[None] < eye_bounds[:,None,2])
-    msk_lon[msk_lon_crosses] |= msk_lon_upper[msk_lon_crosses]
-    msk_lon[msk_lon_within] &= msk_lon_upper[msk_lon_within]
-    return (msk_lat & msk_lon).any(axis=0)
-
-def _vtrans(t_lat, t_lon, t_tstep):
+    msk = (eye_bounds[:,None,1] < centr_lat[None])
+    msk &= (centr_lat[None] < eye_bounds[:,None,3])
+    msk &= (eye_bounds[:,None,0] < centr_lon[None])
+    msk &= (centr_lon[None] < eye_bounds[:,None,2])
+    return msk.any(axis=0)
```

Comment:
Because we are normalizing all longitudinal values, we don't have to consider the antimeridian as a special case here.

Reply:
I added a remark to the docstring of `_close_centroids` for future reference.